hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/server/balancer/BalancingPolicy.java

```diff
@@ -104,21 +104,21 @@ void accumulateSpaces(DatanodeStorageReport r) {
       for(StorageReport s : r.getStorageReports()) {
         final StorageType t = s.getStorage().getStorageType();
         totalCapacities.add(t, s.getCapacity());
-        totalUsedSpaces.add(t, s.getDfsUsed());
+        totalUsedSpaces.add(t, (s.getCapacity() - s.getRemaining()));
```

Comment:
It was not necessary to involve extra outer brackets here.
`totalUsedSpaces.add(t, (s.getCapacity() - s.getRemaining()));` -> `totalUsedSpaces.add(t, s.getCapacity() - s.getRemaining());`